OSDOCS#19124: oc mirror release notes (#111017)
openshift/openshift-docs: modules/rn-ocp-release-notes-new-features.adoc

[id="ocp-release-notes-networking_{context}"]
== Networking

[id="ocp-release-notes-nodes_{context}"]
== Nodes

////
Instructions: Add entries in the following format:

Item description::
+
Detailed information.
////

[id="ocp-release-notes-openshift-cli_{context}"]
== OpenShift CLI (oc)

Digest-based image pinning for the oc-mirror v2 plugin::
+

> With this update, the oc-mirror v2 plugin pins operator catalog images by their digest in your `ImageSetConfiguration` custom resource. Pinning by digest ensures that you always deploy the same operator catalog image, regardless of any later changes to the upstream tags. For more information, see xref:../disconnected/about-installing-oc-mirror-v2.adoc#oc-mirror-workflows-partially-disconnected-v2_about-installing-oc-mirror-v2[Mirroring images for a disconnected installation by using the oc-mirror plugin v2].

Reviewer: ```suggestion
With this update, the oc-mirror v2 plugin pins Operator catalog images by their digest in your `ImageSetConfiguration` custom resource. Pinning by digest ensures that you always deploy the same Operator catalog image, regardless of any later changes to the upstream tags. For more information, see xref:../disconnected/about-installing-oc-mirror-v2.adoc#oc-mirror-workflows-partially-disconnected-v2_about-installing-oc-mirror-v2[Mirroring images for a disconnected installation by using the oc-mirror plugin v2].
```

> Configuration of custom target repositories and tags for additional images by using oc-mirror v2 plugin::

Reviewer: ```suggestion
Configuration of custom target repositories and tags for additional images by using the oc-mirror v2 plugin::
```

Detailed information.
////

[id="ocp-release-notes-openshift-cli_{context}"]
== OpenShift CLI (oc)

Digest-based image pinning for the oc-mirror v2 plugin::
+

> With this update, the oc-mirror v2 plugin pins images by their digest in your `ImageSetConfiguration` custom resource. Pinning by digest maintains consistency across your disconnected environments ensuring that you always deploy the same image, regardless of any later changes to the upstream tags.

Reviewer 2: ```suggestion
With this update, the oc-mirror v2 plugin pins operator catalog images by their digest in your `ImageSetConfiguration` custom resource. Pinning by digest ensures that you always deploy the same operator catalog image, regardless of any later changes to the upstream tags.
```

Configuration of custom target repositories and tags for additional images by using oc-mirror v2 plugin::
+
With this update, when using the oc-mirror v2 plugin, you can provide custom destination repository path and tag for specific images. By using the new `targetRepo` and `targetTag` fields within the `additionalImages` section of your `ImageSetConfiguration` custom resource, you can specify the target repository and tag for an image in your target mirror registry.

Availability of the `oc mirror list` command in oc-mirror v2 plugin::
+

> With this update, when operating in a disconnected environment, you can use the list support feature with the oc-mirror v2 plugin. You can run the `oc mirror list` command to explore available platform and Operator content, including their specific versions, from your registries.

Reviewer 2: ```suggestion
With this update, you can use the list support feature with the oc-mirror v2 plugin. You can run the `oc mirror list` command to explore available platform and Operator content, including their specific versions, from remote and local registries.
```
